.github/workflows/tox-pytest.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         with:
           environment-file: environment.yml
           cache-environment: true
+          cache-environment-key: environment-${{ hashFiles('pyproject.toml') }}
```

Comment:
There was a GDAL conflict with PUDL that I fixed by adding an explicit GDAL requirement, but the CI was still failing because it was using the cached environment, so this will force it to recreate when `pyproject.toml` changes. Also, I've now removed the GDAL thing because Zane fixed it on the PUDL side.